recipes/arboreto/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "arboreto" %}
+{% set version = "0.1.5" %}
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "a78c73b367eaae38dadeba150e52ac98aec6fc916df703ce61b79d27fe1015f9" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.{{ file_ext }}
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+ 
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - scikit-learn
+    - numpy
+    - scipy
+    - pandas
+    - dask
+    - distributed
+
+test:
+  imports:
+    - arboreto
+    - tests
+
+about:
+  home: https://github.com/tmoerman/arboreto
+  license: BSD 3-Clause License
+  license_family: BSD
+  license_file: ''
+  summary: Scalable gene regulatory network inference using tree-based ensemble regressors
+  description: "\
+    \n    *The most satisfactory definition of man from the scientific point of view is probably Man the Tool-maker.*\n\n.. _arboreto: https://arboreto.readthedocs.io\n.. _`arboreto documentation`: https://arboreto.readthedocs.io\n\
```

Comment:
You can remove the description, summary+home is enough, and also easier to maintain in case something changes in the overall description.